yarn.lock

```diff
@@ -1983,7 +1983,7 @@
   dependencies:
     axios "^0.24.0"
 
-"@solidstate/library@^0.0.54":
+"@solidstate/library@^0.0.56":
```

Comment:
I did not run any specific commands to update this. Happy to remove from the PR if needed. Please let me know.